main.swift

```diff
@@ -10,6 +10,12 @@ let languageVersion: String
 
 #if swift(>=5.3)
     #error("Unknown language version")
+#elseif swift(>=5.2.4)
```

Comment:
This actually points out a problem with the code pre-PR! The `#if swift(>=5.3)` up there should have been `#if swift(>=5.2.2)`! After I merge this PR, I'll fix it to be `#if swift(>=5.2.5)`. Thanks!